etc/workflows/delete.xml

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<definition xmlns="http://workflow.opencastproject.org">
+
+  <id>delete</id>
```

Comment:
Okay, so now there are two workflow defintions which have the same content. I think it's not easy to avoid the redundancy here. Also, the two workflow definitions have different use-cases. Maybe someone wants to do additional/other things when deleting an event instead of just retracting it. So this at least gives us more flexibility than before :)